buidl-labs/zombies-in-the-room
# Chapters-view Page - Non-responsive

What does this PR do?
Develop the chapters-view page

Description of Task to be completed?
Implementing the design of the chapters-view page, [design](https://www.figma.com/proto/hcoRbRF5fZREDTJkLl7cXk/BUIDL-for-Tezos?node-id=177%3A2&scaling=scale-down-width)
  

How should this be manually tested?
 > check the preview link generated by netlify build 
 > click on start learning button on homepage, you will see the chapters-view page.
